ReactTask/src/components/CardPatientProfilePatient/CardProfilePatient.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  BoxInfoPatient,
+  BoxDataPatient,
+  StylBoxCardPatient,
+  StylBoxImgPatient,
+  StylBoxColumnInfo,
+  StylBoxRowLine,
+  StylTitleCard,
+  StylSubTitleCardPatient,
+  BtnMore,
+  StylBoxDoctor,
+  StylBoxPatientsList,
+} from '../index.js';
+import heartImg from './../../img/icons/heart.png';
+
+const CardProfilePatient = ({ dataPatients, imgPatientsArr }) => {
+  return (
+    <StylBoxPatientsList>
+      {dataPatients.map((item) => (
+        <StylBoxCardPatient key={item.id}>
+          <StylBoxRowLine>
+            <StylBoxDoctor>
+              <StylBoxImgPatient
+                src={imgPatientsArr[item.id - 1]}
+                alt={item.alt}
+              />
+              <StylBoxColumnInfo>
+                <StylTitleCard>{item.name}</StylTitleCard>
+                <StylSubTitleCardPatient>
+                  {item.statusText}
+                </StylSubTitleCardPatient>
+              </StylBoxColumnInfo>
+            </StylBoxDoctor>
+            <BtnMore />
+          </StylBoxRowLine>
+          <BoxDataPatient data={item.data} />
+          <BoxInfoPatient info={item.info} imgWay={heartImg} />
```

Comment:
I think it would be easier for you to make this picture as a background. You won't interact with it. This is purely my opinion

Reply:
Used `:after`

Reply:
Resolved

Reply:
What's purpose of  importing picture in component, then via props u throw this img in 3more components???
Why just not import in Boxes.styled.js?

Reply:
I have the `StylBoxDoctor` class  used in several places and in some cases I need to insert a picture. For this reason, I use a props and draw as I please.

Reply:
Resolved